src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolver.cs

```diff
@@ -152,21 +152,24 @@ internal override string GetRequestModuleName(DkmRuntimeFunctionResolutionReques
             return request.ModuleName;
         }
 
-        internal override void OnFunctionResolved(
-            DkmClrModuleInstance module,
-            DkmRuntimeFunctionResolutionRequest request,
-            int token,
-            int version,
-            int ilOffset)
-        {
-            var address = DkmClrInstructionAddress.Create(
-                module.RuntimeInstance,
-                module,
-                new DkmClrMethodId(Token: token, Version: (uint)version),
-                NativeOffset: uint.MaxValue,
-                ILOffset: (uint)ilOffset,
-                CPUInstruction: null);
-            request.OnFunctionResolved(address);
+        private static OnFunctionResolvedDelegate<DkmClrModuleInstance, DkmRuntimeFunctionResolutionRequest> OnFunctionResolved(DkmWorkList workList)
+        {
+            return (DkmClrModuleInstance module,
+                        DkmRuntimeFunctionResolutionRequest request,
+                        int token,
+                        int version,
+                        int ilOffset) =>
+            {
+                var address = DkmClrInstructionAddress.Create(
+                    module.RuntimeInstance,
+                    module,
+                    new DkmClrMethodId(Token: token, Version: (uint)version),
+                    NativeOffset: 0,
```

Comment:
`NativeOffset:` was incorrect previously, the value should be 0 rather than -1, although it's not clear if the difference is observable.